content/departments/cloud/technical-docs/v2.0/creation_process.md

```diff
@@ -16,13 +16,21 @@ cd cloud
 
 > See flow chart https://app.excalidraw.com/s/4Dr1S6qmmY7/2wUSD4kIxRo
 
-1. [Set environment variables](#Set-environment-variables)
-1. [Check out a new branch](#Check-out-a-new-branch)
-1. [Init deployment artifacts - GCP Project](#init-deployment-artifacts---gcp-project)
-1. [Init deployment artifacts - Infrastructure](#init-deployment-artifacts---infrastructure)
-1. [Init deployment artifacts - K8S](#init-deployment-artifacts---k8s)
-1. [Deploy application](#deploy-application)
-1. [Commit your changes](#Commit-your-changes)
+- [Creating a managed instance](#creating-a-managed-instance)
+  - [Prereq](#prereq)
+  - [Steps](#steps)
+    - [Set environment variables](#set-environment-variables)
+    - [Check out a new branch](#check-out-a-new-branch)
+    - [Init deployment artifacts - GCP Project](#init-deployment-artifacts---gcp-project)
+    - [Init deployment artifacts - Infrastructure](#init-deployment-artifacts---infrastructure)
+    - [Init deployment artifacts - K8S](#init-deployment-artifacts---k8s)
+    - [Deploy application](#deploy-application)
+    - [Commit your changes](#commit-your-changes)
+  - [Troubleshooting](#troubleshooting)
+    - [PVC is stuck at `Pending`](#pvc-is-stuck-at-pending)
+    - [How do I check my Sourcegraph deployment rollout progress?](#how-do-i-check-my-sourcegraph-deployment-rollout-progress)
+    - [I am unable to apply the terraform module due to permission error.](#i-am-unable-to-apply-the-terraform-module-due-to-permission-error)
+    - [Any other questions?](#any-other-questions)
```

Comment:
can we revert the change here?

this section is meant to provide an overview of the creation process instead of ToC of the entire page, and there's already a built-in ToC on the right. also, I found it really hard to maintain the ToC by hand.

![CleanShot 2022-09-28 at 09 45 41](https://user-images.githubusercontent.com/8373004/192841278-18b12d10-2ed9-4d3d-a35f-ce4d6b8eda83.png)

Reply:
Ah this was an automated update, reverted